packages/services/src/kernel/messages.ts

```diff
@@ -1256,3 +1256,14 @@ export interface IInputReplyMsg extends IStdinMessage<'input_reply'> {
 export function isInputReplyMsg(msg: IMessage): msg is IInputReplyMsg {
   return msg.header.msg_type === 'input_reply';
 }
+
+// ///////////////////////////////////////////////
+// Message (de)serialization
+// ///////////////////////////////////////////////
+
+/**
+ * The list of supported kernel wire protocols over websocket.
+ */
+export const supportedKernelWebSocketProtocols: string[] = [
+  'v1.kernel.websocket.jupyter.org'
+];
```

Comment:
Could you switch to a [enumeration](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) with text value? This will clarify the code.

Reply:
:+1: